src/Assignment4/SearchComponent/index.js

```diff
@@ -0,0 +1,36 @@
+import "./styles.css";
+
+import { useState } from "react";
```

Comment:
Imports from global modules should be on the top. Isn't it?

Reply:
done